jwala-webservices/src/main/java/com/cerner/jwala/ws/rest/v1/service/group/impl/GroupServiceRestImpl.java

```diff
@@ -705,10 +712,12 @@ public Response generateAndDeployGroupAppFile(final String groupName, final Stri
                 resourceService.validateSingleResourceForGeneration(resourceIdentifier);
                 if (hostName != null && !hostName.isEmpty()) {
                     // deploy to particular host
+                    historyFacadeService.write(hostName, group, "Deploying application resource " +
+                            resourceIdentifier.resourceName, EventType.USER_ACTION_INFO, aUser.getUser().getId());
                     performGroupAppDeployToHost(groupName, fileName, appName, hostName, metaData.isHotDeploy());
                 } else {
                     // deploy to all hosts in group
-                    performGroupAppDeployToHosts(groupName, fileName, appName, metaData.isHotDeploy());
+                    performGroupAppDeployToHosts(groupName, fileName, appName, metaData.isHotDeploy(), aUser);
```

Comment:
This is where the resource is deployed to all JVMs in a group. Write a message here to the history stating the resource is being deployed to all JVMs in the group.